docs/overview/related_work/api_standards.rst

```diff
@@ -16,8 +16,24 @@ This standard can be considered as “higher level” than the ML frameworks the
 The Array API Standard takes the lowest common denominator approach, whereby each function in the standard represents the minimum behaviors of the function without restricting extensions to the function.
 This means that two very different libraries can adhere to the same standard, despite having very different extended behaviors for some of the functions in the standard.
 The standard is also not exhaustive.
-For example, there are <insert_number> functions defined in the standard, whereas the functions defined in each framework are as follows:
+For example, there are 64 functions defined in the standard, whereas the functions defined in each framework are as follows:
 
-<insert_table>
+Table:
```

Comment:
Some clarification points regarding this 
- the table is not being rendered correctly in the `rst` format. Could you kindly look into this? 
- Also, I am not sure what the content in this table represents. Upon doing a manual count, there seem to be a total of 51 functions in the table whereas you've mentioned 64 functions in the Array API standard (could you also link a source from where you've accessed this information). This seems counter-intuitive to the above statement: 
> The standard is also not exhaustive
- Instead of `Pandas`, its better to stick to DL frameworks such as `JAX` or `PaddlePaddle`


PS: Actually I think I now understand what the table intends to represent.  I think the table is supposed to represent the functions that **are not** defined in the standard, yet are part of the Functional API of these frameworks. In this case, the table should also reflect that. Perhaps you should add a title or description to the table to make it clear that these are some of the functions that **are not** in the Array API Standard